src/main/java/com/google/api/codegen/transformer/StaticLangApiMethodTransformer.java

```diff
@@ -454,7 +464,11 @@ public StaticLangApiMethodView generateAsyncOperationFlattenedMethod(
     methodViewBuilder.exampleName(namer.getAsyncApiMethodExampleName(method));
     methodViewBuilder.callableName(namer.getCallableName(method));
     setFlattenedMethodFields(
-        context, additionalParams, Synchronicity.Async, methodViewBuilder, Arrays.asList(type));
+        context,
+        additionalParams,
+        Synchronicity.Async,
+        methodViewBuilder,
+        Arrays.asList(CallingForm.LongRunningFlattenedAsync));
```

Comment:
Similar to above: how do we know this is always flattened and async?

Reply:
We're in method `generateAsyncOperationFlattenedMethod`, so I'm taking the `Async` and `Flattened` at face value, and interpreting `Operation` as `LongRunning`. Are any of these inferred conventions incorrect? If so, we can trace the logic more thoroughly.